### DISCUSSION TITLE
How to index existing photo directly

### DISCUSSION THREAD

**Discussion by sainiaj** [photoprism/photoprism#2995] [Q&A]
Hi,

I would like to see/index existing photos only,

Let's say all my photos are located and mounted in /mnt/Photos

I am using Proxmox PVE container added using a script - https://tteck.github.io/Proxmox/

I am not able to see any of my photos or the folders inside the Photoprism not sure where to look correctly, 

I can ssh and see the mounted folder photos folder - so permissions are correct, I would just like to know how I can index them

I would like to add the main folder Photos and let Photoprism do its magic and index them all recursively 

Thank You.

**Comment by graciousgrey** [comment]
Our first steps guide explains how to add files --> https://docs.photoprism.app/user-guide/first-steps/

We also do provide troubleshooting checklists in case there are problems with your setup: https://docs.photoprism.app/getting-started/troubleshooting/

If those don't help please contact the proxmox community as we can't help much with third party installation problems.
